.github/workflows/nightly_run.yaml

```
@@ -0,0 +1,76 @@
+name: Test Update smoke 
+"on":
+  # Going to run the scripts/test_update_smoke.sh once each night at midnight
+  # Currently staying with Postgres 17
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+  pull_request:
```

We probably don't want to run this for every pull request, so for development, you can use this:

```suggestion
  pull_request:
    paths:
      - .github/workflows/nightly_run.yaml
```

---

Again, thanks for the suggestion!
I've updated the file.